packages/helper-grammar-regex-collection/test.js

```diff
@@ -387,12 +387,33 @@ const fixtures = {
         'import (\n"golang.org/x/net/context"\n"golang.org/pkg/net"\n)',
         ['golang.org/x/net/context', 'golang.org/pkg/net'],
       ],
+
+      // go.mod
+      ['require github.com/foo v1.1.0', ['github.com/foo']],
+      ['require bitbucket.org/foo v1.1.0', ['bitbucket.org/foo']],
+      ['require launchpad.net/foo v1.1.0', ['launchpad.net/foo']],
+      ['require    github.com/foo v1.1.0', ['github.com/foo']],
+      ['require github.com/foo v1.1.0      ', ['github.com/foo']],
+      ['require github.com/foo v1.1.0\n', ['github.com/foo']],
+
+      ['require github.com/foo/v2', ['github.com/foo/v2']],
+
+      ['require (\ngithub.com/foo v1.1.0\n)', ['github.com/foo']],
+      [
+        'require (\ngithub.com/foo v1.1.0\n\ngithub.com/bar v1.1.0\n)',
+        ['github.com/foo', 'github.com/bar'],
+      ],
     ],
     invalid: [
       'simport foo',
       'simport\nfoo',
       'import "octo.com/foo/bar"',
       'import (\n"octo.com/foo/bar"\n)',
+
+      'require (github.com/foo v1.1.0)',
+      'require \n"foo" v1.1.0',
+      'srequire "foo" v1.1.0',
```

Comment:
@stefanbuck looks like the npm parser is picking these two lines up.

![image](https://user-images.githubusercontent.com/831974/76724347-1315d100-6721-11ea-8fa8-4e6efad67bae.png)

Reply:
How do you know?

Reply:
It's picking up `foo` and linking it to https://github.com/jasonLaster/foo. It's not an issue with this PR, that's something with the current npm parser that needs to be looked at.

Reply:
@xt0rted good spot, but I wonder if its worth a fix. The file extension is `.js` and therefore the `require` regexp matches this string as well.